test/tests.js

```diff
@@ -11,10 +22,275 @@ test('this test should pass', (expect) => {
+    const expected2 = 14;
+
+    const actual2 = multiplyBySeven(2);
+
+    expect2.deepEqual(actual2, expected2, '2 * 7 = 14');
+});
+
+test('This function should take in a number and multiply it by seven', (expect3) => {
+    const expected3 = 21;
+
+    const actual3 = multiplyBySeven(3);
+
+    expect3.deepEqual(actual3, expected3, '3 * 7 = 21');
+});
+
+test('This function should take in a number and multiply it by twelve then cut the result in half', (expect) => {
+    const expected = 24;
+
+    const actual = multiplyBy12ThenHalve(4);
+
+    expect.deepEqual(actual, expected, '(4 * 12)/2 = 24');
+});
+
+test('This function should take in a number and multiply it by twelve then cut the result in half', (expect2) => {
+    const expected2 = 12;
+
+    const actual2 = multiplyBy12ThenHalve(2);
+
+    expect2.deepEqual(actual2, expected2, '(2 * 12)/2 = 12');
+});
+
+test('This function should take in a number and multiply it by twelve then cut the result in half', (expect3) => {
+    const expected3 = 18;
+
+    const actual3 = multiplyBy12ThenHalve(3);
+
+    expect3.deepEqual(actual3, expected3, '(3 * 12)/2 = 18');
+});
+
+test('This function should take in three numbers, divide the first by the second, then multiply the result by the third', (expect) => {
+    const expected = 10;
+
+    const actual = divideThenMultiply(8, 4, 5);
+
+    expect.deepEqual(actual, expected, '(8/4)5 = 10');
+});
+
+test('This function should take in three numbers, divide the first by the second, then multiply the result by the third', (expect2) => {
+    const expected2 = 50;
+
+    const actual2 = divideThenMultiply(20, 2, 5);
+
+    expect2.deepEqual(actual2, expected2, '(20/2)5 = 50');
+});
+
+test('This function should take in three numbers, divide the first by the second, then multiply the result by the third', (expect3) => {
+    const expected3 = 100;
+
+    const actual3 = divideThenMultiply(40, 2, 5);
+
+    expect3.deepEqual(actual3, expected3, '(40/2)5 = 50');
+});
+
+test('This function should take in three numbers and return those numbers in an array', (expect) => {
+    const expected = [8, 4, 5];
+
+    const actual = returnAsAnArray(8, 4, 5);
+
+    expect.deepEqual(actual, expected, '8,4,5 should be [8, 4, 5]');
+});
+
+test('This function should take in three numbers and return those numbers in an array', (expect2) => {
+    const expected2 = [1, 2, 3];
+
+    const actual2 = returnAsAnArray(1, 2, 3);
+
+    expect2.deepEqual(actual2, expected2, '1,2,3 should be [1, 2, 3]');
+});
+
+test('This function should take in three numbers and return those numbers in an array', (expect3) => {
+    const expected3 = [3, 4, 5];
+
+    const actual3 = returnAsAnArray(3, 4, 5);
+
+    expect3.deepEqual(actual3, expected3, '3,4,5 should be [3, 4, 5]');
+});
+
+test('This function should take in three numbers and return those numbers mushed together as a string', (expect) => {
+    const expected = '8, 4, 5';
```

Comment:
The expected output here should be a string of the numbers mushed together. So no commas.